beefy-primitives/src/lib.rs

```diff
@@ -80,7 +80,7 @@ pub type MmrRootHash = H256;
 pub enum ConsensusLog<AuthorityId: Codec> {
 	/// The authorities have changed.
 	#[codec(index = 1)]
-	AuthoritiesChange(Vec<AuthorityId>),
+	AuthoritiesChange((Vec<AuthorityId>, ValidatorSetId)),
```

Comment:
```suggestion
	AuthoritiesChange(Vec<AuthorityId>, ValidatorSetId),
```

Instead of having one-value enum with a tuple instead, we can just do two-valued enum

Reply:
You can also consider naming the parameters to add some extra docs:
```rust
AuthoritiesChange {
   /// ...
   new_validator_set: Vec<AuthorityId>,
   /// ...
   new_validator_set_id: ValidatorSetId,
}
```